src/util/atoms.ts

```diff
@@ -15,6 +15,7 @@ export const bridgeConfigAtom = atom<BridgeConfig>({
   SBTC_CONTRACT_DEPLOYER: undefined,
   SIGNER_AGGREGATE_KEY: undefined,
   BANNER_CONTENT: undefined,
+  RECLAIM_LOCK_TIME: 144,
```

Comment:
Why is the default duplicated here and in `env.ts`?

Reply:
env.ts is only accessible to the server

Reply:
Removed duplication, added a fallback in `DepositFlowConfirm` in case env var is ever undefined